core/data/src/main/java/com/tgyuu/data/repository/ConfigRepositoryImpl.kt

```
@@ -96,4 +109,24 @@ class ConfigRepositoryImpl @Inject constructor(
+            runCatching {
+                val uuid = localSyncDataSource.uuid.first()
+                if (uuid == "INVALID") return@launch
+
+                val enabled = localUserConfigDataSource.notificationEnabled.first()
+                val (hour, minute) = localUserConfigDataSource.alarmTime.first()
+                val message = localUserConfigDataSource.alarmMessage.first()
+
+                notificationLogDataSource.logNotificationConfig(
+                    uuid = uuid,
+                    enabled = enabled,
+                    alarmTime = String.format("%02d:%02d", hour, minute),
```

detekt (1.23.8)

[warning] String.format("%02d:%02d", hour, minute) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)